examples/image_reconstruction.ipynb

```diff
@@ -0,0 +1,705 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "CMdgh_Jmw_Pu"
+   },
+   "source": [
+    "<div class=\"alert alert-success\">\n",
+    "    <h1 align=\"center\">Image reconstruction</h1>\n",
+    "    <h1 align=\"center\">Using the Poutyne framework</h1>\n",
+    "    <h3 align=\"center\"><a href=\"https://poutyne.org/\"> Click on this tab for more information</a></h3>\n",
+    "</div>\n"
```

Comment:
```suggestion
    "# Image reconstruction using Poutyne"
```
I would prefer we do not use HTML in notebooks so that we stay consistent with the other one. Just a simple markdown title will do.